clients/privacy-center/components/Card.tsx

```diff
@@ -8,6 +11,40 @@ type CardProps = {
   onClick: () => void;
 };
 
+/**
+ * IconProviders maps to the react-icons libraries supported by Fides.
+ * The values of the enum elements are the prefix to indicate that the iconPath should use an icon library instead of a relative path or URL.
+ */
+enum IconProviders {
+  MATERIALDESIGN = "md",
+  FONTAWESOME = "fa",
+}
+
+function iconResolver(iconPath: string, description: string): JSX.Element {
```

Comment:
this is a nice feature! I think it would make more sense as a full on component rather than a function in `Card.tsx` though. Perhaps a component called `ResolvedIcon` which takes the same props you're passing into `iconResolver`, and it could go in its own file `ResolvedIcon.tsx`. The inner part would look largely the same, but it would be more React-y (and also we'd be able to use it in another places!)